lib/scsi_defs.h

```diff
@@ -80,5 +85,21 @@ namespace scsi_defs {
     uint8_t vendorSpecific : 2;
   } ControlByte;
 
+  //SCSI Reference Manual Table 202 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  typedef struct TestUnitReadyCommand {
+    OpCode opCode = OpCode::kTestUnitReady;
+    uint32_t reserved;
+    ControlByte controlByte;
+    uint16_t padding;   // Necessary for bit_cast
+
+    static std::optional<TestUnitReadyCommand> fromBits(std::bitset<48> cmd) {
```

Comment:
I am working on next steps, and one would be parser and handler of raw data. I would prefer to have all parsing and handling in a single place instead of having it across all the structures.

Reply:
So instead of having "constructors" for each class, we handle that logic elsewhere?